libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -36,6 +40,7 @@ public class EntitlementInitialization {
 
     private static final Module ENTITLEMENTS_MODULE = PolicyManager.class.getModule();
 
+    public static InitializeArgs initializeArgs;
```

Comment:
`InitialiationArgs`?

Reply:
I figured they are the arguments for the `initialize` method. Not the best name I ever invented.